.github/codeql/codeql-config.yml

```diff
@@ -0,0 +1,26 @@
+name: "ospac-simcore CodeQL config"
+
+disable-default-queries: false
```

Comment:
So what are we scanning for by default?

Reply:
Check the links above RT... :-)  ... but basically thinks like plotting sensitive info (e.g. he was able to trace in the code and detect that a password was printed in the logs ... was quite impressive BTW)